### ISSUE TITLE
NTR: CT or MRI: deep neck abscess (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9017]
New term request CT or MRI: deep neck abscess (nadea3_190131013910): Retropharyngeal abscess, peritonsillar abscess, RPA, PTA; cervical  UMLS: 0702176

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000288
- label: Deep neck abscess
- parents: [Abscess (HP:0025615)](https://hpo.jax.org/app/browse/term/HP:0025615)
- pmid: PMID:30020634
- def: An abscess (i.e., a circumscribed area of pus or necrotic debris) localized in the area surrounded by the deep fascia of the neck including the pharynx, larynx, trachea, upper esophagus, thyroid, and parathyroid glands. Infections of pharyngeal, tonsillar, and laryngeal origin affect this space, which includes the parapharyngeal and retropharyngeal spaces.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9017
- synonyms:
